convene-web/app/views/rooms/show.html.erb

```diff
@@ -13,6 +13,20 @@
 >
 </div>
 
+<%- room.furniture_placements.each do |furniture_placement| %>
+  <div
+    class="<%=furniture_placement.furniture_slug%>"
+    data-controller="<%=furniture_placement.furniture_slug %>"
+    <%- furniture_placement.data_tuples.each do |name, value| %>
```

Comment:
What are these `data_tuples`? Is that an intended field on `FurniturePlacement` that is currently missing from the model, or something that Rails magically provides? (Quick search for this didn't shed light, sadly.)

Reply:
My intent was for this to be a way to expose data to stimulus: https://stimulusjs.org/handbook/managing-state#the-data-api-explained. It's a bit of "coding by wishful thinking" in that I never actually defined them; but essentially they are a mechanism for a piece of furniture to expose data to the front-end.

Reply:
I've ditched this in favor of each piece of Furniture being able to define it's own partial, so that we're not coupling implementation details of how we pass data to the UI too early..